modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/raft/SnapshotMetaResponse.java

```diff
@@ -20,11 +20,10 @@
 import org.apache.ignite.internal.network.NetworkMessage;
 import org.apache.ignite.internal.network.annotations.Transferable;
 import org.apache.ignite.internal.partition.replicator.network.PartitionReplicationMessageGroup;
-import org.apache.ignite.raft.jraft.entity.RaftOutter.SnapshotMeta;
 
 /** Snapshot meta response message. */
 @Transferable(PartitionReplicationMessageGroup.SNAPSHOT_META_RESPONSE)
 public interface SnapshotMetaResponse extends NetworkMessage {
     /** Snapshot meta. */
-    SnapshotMeta meta();
+    PartitionSnapshotMeta meta();
```

Comment:
I think we also need to rename the request and response classes to show that they are only applicable to partitions

Reply:
It's in a package related to partitions, so it seems to be already clear from the context. WDYT?

Reply:
Up to you, I won't insist